design/sketch.webidl

```diff
@@ -627,14 +627,13 @@ interface GPUQueue {
 
 // SwapChain / RenderingContext
 dictionary GPUSwapChainDescriptor {
-    GPUTextureUsageFlags usage;
-    GPUTextureFormat format;
+    required GPUCanvasContext context;
+    required GPUTextureFormat format;
+    GPUTextureUsageFlags usage = GPUTextureUsage.OUTPUT_ATTACHMENT;
 };
 
 interface GPUSwapChain {
-    void configure(GPUSwapChainDescriptor descriptor);
-    GPUTexture getNextTexture();
-    void present();
+    GPUTexture getTexture();
 };
 
 interface GPURenderingContext : GPUSwapChain {
```

Comment:
we need to remove this

Reply:
@kvark , can you please elaborate why you would like `getTexture` to be removed?  How else would the web developer determine where to draw?